src/routes/payments.py

```
@@ -68,3 +73,18 @@
     return CreatePaymentSessionResponseSchema(
         session_url=cast(AnyUrl, session_url)
     )
+
+
+@router.post("/stripe/webhook")
+async def handle_stripe_webhook_event(
+    request: Request,
+    settings: BaseAppSettings = Depends(get_settings),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)